<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TinyMTSeedFinder</class>
 <widget class="QWidget" name="TinyMTSeedFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TinyMT Seed Finder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="8">
    <widget class="QListWidget" name="blinkList"/>
   </item>
   <item row="1" column="9">
    <widget class="QPushButton" name="cancelSearchButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cancel Search</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="10">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="TextBox" name="maxAdvanceTextBox"/>
   </item>
   <item row="0" column="8" colspan="2">
    <widget class="QListWidget" name="resultList"/>
   </item>
   <item row="6" column="2">
    <widget class="TextBox" name="minAdvanceTextBox"/>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="startSearchButton">
     <property name="text">
      <string>Start Search</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Advance Range</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="TextBox" name="maxSeedTextBox"/>
   </item>
   <item row="5" column="2">
    <widget class="TextBox" name="minSeedTextBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Seed Range</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>~</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>~</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="addBlinkButton">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="text">
      <string>Start Recording Blinks</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="5">
    <widget class="QPushButton" name="restartBlinksButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Restart</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
